### PR TITLE
Change har snippets

### PR DESCRIPTION
**Pull request by khorne3**

### CODE REVIEW

File: articles/tutorials/calling-an-external-idp-api.md [hunk 1]
```diff
@@ -51,15 +51,22 @@ The following code executes a client credentials exchange so that you receive an
...
-  "method": "POST",
-  "url": "https://${account.namespace}/oauth/token",
-  "headers": [
-    { "name": "Content-Type", "value": "application/json" }
-  ],
-  "postData":{
-    "mimeType": "application/json",
-    "text": "{ \"client_id\": \"${account.clientId}\", \"client_secret\": \"${account.clientSecret}\", \"audience\": \"https://${account.namespace}/api/v2/\", \"grant_type\": \"client_credentials\" }"
-  }
+	"method": "POST",
```
Comment (mpaktiti) [inline articles/tutorials/calling-an-external-idp-api.md]: @khorne3 I'm pretty sure this is wrong, and the way it was before was the correct one. The valid way to get an APIv2 token is described here https://auth0.com/docs/api/management/v2/tokens#1-get-a-token

I even tried to get a token with the new way described here but if I don't specify the `audience` param, the token I'm getting is an opaque token, not a JWT, and if I try to use it at the Management APIv2 I get the following error:

image

Did someone from engineering give this input or can I fix it and merge?

**Review by mpaktiti — CHANGES_REQUESTED** [review]
The described way to get an APIv2 token does not seem correct.

File: articles/tutorials/calling-an-external-idp-api.md (hunk 1 above)
Comment (khorne3) [reply articles/tutorials/calling-an-external-idp-api.md]: You can fix it and merge; thanks!

**Comment by khorne3** [comment]
@mpaktiti Reverted.

**Comment by mpaktiti** [comment]
@khorne3 I just realized there are more changes required.

The process to get a Management APIv2 Token has changed. You can do it either using client credentials or using the dashboard. All the details are [here](https://auth0.com/docs/api/management/v2/tokens).

I would leave 4 paragraphs/steps:
- __Step 1: Create a Client__ (add some of the info [here](https://auth0.com/docs/api/management/v2/tokens#1-create-and-authorize-a-client))
- __Step 2: Obtain an Auth0 Access Token__ (either [copy it from the dashboard](https://auth0.com/docs/api/management/v2/tokens#2-get-the-token) or [automate the process using client credentials](https://auth0.com/docs/api/management/v2/tokens#automate-the-process)).
- __Step 3: Obtain the User Profile__ (same content as https://auth0-docs-content-pr-3659.herokuapp.com/docs/tutorials/calling-an-external-idp-api#2-obtain-the-user-profile)
- __Step 4: Extract the idP Access Token__ (same content as https://auth0-docs-content-pr-3659.herokuapp.com/docs/tutorials/calling-an-external-idp-api#3-extract-the-idp-access-token)

**Pull request merged**